core-common/src/main/kotlin/com/twitter/rules/core/util/Composables.kt

```diff
@@ -146,3 +146,15 @@ private val CompositionLocalReferenceExpressions by lazy(LazyThreadSafetyMode.NO
         "compositionLocalOf"
     )
 }
+
+val KtCallExpression.isRestartableEffect: Boolean
+    get() = RestartableEffects.contains(calleeExpression?.text)
+
+// From https://developer.android.com/jetpack/compose/side-effects#restarting-effects
+private val RestartableEffects by lazy(LazyThreadSafetyMode.NONE) {
+    setOf(
+        "LaunchedEffect",
```

Comment:
Not the list of restartable effects per se, but it would be nice to add them as an extra list of things allowed in ViewModelForwarding (eg you have a composable that uses the VM for keying). Can't think of an use case on the top of my head but I'm sure it could be useful to support. And given that the cost of adding it to the config is minimal, I'll do that 😆